internal/cephes/incbeta.go

```diff
@@ -0,0 +1,312 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+ * Cephes Math Library, Release 2.3:  March, 1995
+ * Copyright 1984, 1995 by Stephen L. Moshier
+ */
+
+package cephes
+
+import (
+	"math"
+
+	"github.com/gonum/mathext/internal/gonum"
+)
+
+const (
+	maxGam = 171.624376956302725
+	big    = 4.503599627370496e15
+	biginv = 2.22044604925031308085e-16
+)
+
+// Regincbet computes the regularized incomplete beta function.
+func Incbet(aa, bb, xx float64) float64 {
```

Comment:
I meant the mismatch beyween the function and the comment. Following the library is ok.